tools/scripts/generate_binary_build_matrix.py

```diff
@@ -66,8 +66,10 @@ def validation_runner(arch_type: str, os: str) -> str:
             return WIN_CPU_RUNNER
     elif os == "macos-arm64":
         return MACOS_M1_RUNNER
-    elif os == "macos-x86_64":
-        return MACOS_X86_RUNNER
+    elif os == "macos":
```

Comment:
@weiwangmeta might have an opinion here since he added these

Reply:
I guess it will depend on the parameters we are use to call: https://github.com/pytorch/test-infra/pull/656/files#diff-5dbd10d92a7f7b873a48853f3ac5cae96d055c09d25e1734005eb0ce11e11aebR15

I did not find any other places 👀